R/zzz.R

```diff
@@ -73,23 +72,23 @@
                     assignEnumCallback = .assignEnumCallback,
                     functionFilter = .cherryPickTableFunctionFilter,
                     classFilter = .removeAllClassesClassFilter,
-                    functionPrefix = "syn",
-                    transformReturnObject = .objectDefinitionHelper)
+                    functionPrefix = "syn")
 }
 
-.objectDefinitionHelper <- function(object) {
-  if (methods::is(object, "CsvFileTable")) {
-    # reading from csv
-    # Removed due to Error in unlockBinding("asDataFrame", object) : no binding for "asDataFrame"
-    # unlockBinding("asDataFrame", object)
-    object$asDataFrame <- function() {
-      .readCsvBasedOnSchema(object)
-    }
-    # Removed due to Error in lockBinding("asDataFrame", object) : no binding for "asDataFrame"
-    # lockBinding("asDataFrame", object)
-  }
-  object
-}
+
+# .objectDefinitionHelper <- function(object) {
```

Comment:
If this is code we want to keep for later then it's fine to comment it out - Though it should have a TODO comment on what should be done with it.

If we are going to remove the code totally, then I don't see a need to keep the commented out code.

Reply:
Sure I can add TODO comments for it. This is the chunk of code cause the infinite recursion issue when reading downloaded entity to dataframe. I prefer to keep this for now so we can double check on it when we doing the deep cleaning for synapser.

Reply:
TODO comments added.